Answers.md

```diff
@@ -1,5 +1,10 @@
 1. What problem does the context API help solve?
+A: More accessible way of managing the state of the APP.
 1. In your own words, describe `actions`, `reducers` and the `store` and their role in Redux. What does each piece do? Why is the store known as a 'single source of truth' in a redux application?
+A: actions and reducers are both for managing the state and the data logic in APP, i.e., in actions file you can create functions to make an api call and on reducers you can export those functions as a separate cases.
 1. What is the difference between Application state and Component state? When would be a good time to use one over the other?
+A: I think the state on both of them are pretty similar.
```

Comment:
Difference is that application state is global and available throughout the app. Component state is available to that component.